packages/legend-query/src/stores/QueryBuilderSetupState.ts

```diff
@@ -211,4 +217,16 @@ export class QueryBuilderSetupState {
         )
       : val;
   }
+
+  *setModelCoverageAnalyticsResult(): GeneratorFn<void> {
```

Comment:
--> `analyzeMappingModelCoverage`. Also, let's move `refreshTreeData` to inside this method when the call succeeds